File: src/v6/Person.java
package v6;

public class Person {
    private static int nextId=1;
    final int id;
    private  String name;
    private  String nickname;
    private  String phoneNumber;
    private  String address;
    private String email;
    private String gender;

    public Person(String name,
      String nickname,
      String gender,
      String phoneNumber,
      String address,
      String email) {

        this.id=nextId++;
        this.name=name;
        this.address=address;
        this.nickname=nickname;
        this.phoneNumber=phoneNumber;
        this.email=email;
        this.gender=gender;

    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public String getAddress() {
        return address;
    }

    public String getName() {
        return name;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public int getId() {
        return id;
    }

    public String getEmail() {
        return email;
    }

    public String getNickname() {
        return nickname;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public void setNickname(String nickname) {
        this.nickname = nickname;
    }
}

File: pythonCodeConversion/electricbill/models/person.py
class Person:
    nextId = 1

    def __init__(self, name, nickname, gender, phone_number, address, email):
        self.id = Person.nextId
        Person.nextId += 1
        self.name = name
        self.address = address
        self.nickname = nickname
        self.phoneNumber = phone_number
        self.email = email
        self.gender = gender

    def get_gender(self):
        return self.gender

    def set_gender(self, gender):
        self.gender = gender

    def get_address(self):
        return self.address

    def get_name(self):
        return self.name

    def get_phone_number(self):
        return self.phoneNumber

    def get_id(self):
        return self.id

    def get_email(self):
        return self.email

    def get_nickname(self):
        return self.nickname

    def set_address(self, address):
        self.address = address

    def set_name(self, name):
        self.name = name

    def set_phone_number(self, phone_number):
        self.phoneNumber = phone_number

    def set_email(self, email):
        self.email = email

    def set_nickname(self, nickname):
        self.nickname = nickname
